### PR TITLE
feat: update staff media list toggle to menu

### PR DESCRIPTION
**Pull request by BitForger**
I couldn't get the alignment to work right. I wanted to to float the label to the left and the picker to the right but it wasn't doing what I wanted....
CleanShot 2024-02-05 at 20 57 08@2x

### CODE REVIEW

File: AniHyou/Screens/StaffDetails/ViewModels/Enums.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+//  Enums.swift
+//  AniHyou
+//
+//  Created by Noah Kovacs on 2/5/24.
+//
+
+import Foundation
+import SwiftUI
+
+public enum StaffOnList: String, Equatable, CaseIterable {
```
Comment (axiel7) [inline AniHyou/Screens/StaffDetails/ViewModels/Enums.swift:11]: rename it to `OnList` because we'll also use it for characters and search

Comment (axiel7) [inline AniHyou/Screens/StaffDetails/ViewModels/Enums.swift:11]: I already had a picker for selecting three states, use it. [TriPicker](https://github.com/axiel7/AniHyou-iOS/blob/main/AniHyou/Views/TriPicker.swift)

File: AniHyou/Screens/StaffDetails/Views/StaffDetailsView.swift [hunk 2]
```diff
@@ -110,7 +110,15 @@ struct StaffDetailsView: View {
     @ViewBuilder
     var staffMedia: some View {
         LazyVStack(alignment: .leading) {
```
Comment (axiel7) [inline AniHyou/Screens/StaffDetails/Views/StaffDetailsView.swift:112]: for fixing the picker width, try to change `LazyVStack` to a `List` and apply the `.listStyle(.plain)` to it.

File: AniHyou/Screens/StaffDetails/Views/StaffDetailsView.swift [hunk 3]
```diff
@@ -110,7 +110,15 @@ struct StaffDetailsView: View {
...
-            Toggle("On my list", isOn: $viewModel.mediaOnMyList)
+            HStack {
```
Comment (axiel7) [inline AniHyou/Screens/StaffDetails/Views/StaffDetailsView.swift:113]: no need to create a HStack with an extra Text. Use just the picker.

Comment (BitForger) [reply AniHyou/Screens/StaffDetails/Views/StaffDetailsView.swift:113]: I'll try using that picker you created.. the `Picker` element wasn't putting the label on the screen for some reason which is why I used that stack

**Pull request merged**